src/vs/workbench/contrib/chat/browser/chatContentParts/toolInvocationParts/chatTerminalToolProgressPart.ts

```diff
@@ -148,6 +152,18 @@ export class ChatTerminalToolProgressPart extends BaseChatToolInvocationSubPart
 
 		const progressPart = this._register(_instantiationService.createInstance(ChatProgressSubPart, elements.container, this.getIcon(), terminalData.autoApproveInfo));
 		this.domNode = progressPart.domNode;
+		this.domNode.tabIndex = 0;
```

Comment:
Isn't it bad practice/UX to make something focusable when it can't actually be interacted with?

Reply:
It can be interactive though - does this link not exist anymore?

<img width="293" height="43" alt="Screenshot 2025-10-31 at 12 40 21 PM" src="https://github.com/user-attachments/assets/0d4d5def-7cfb-45e7-b8ce-9114a98161d3" />